src/ansible_navigator/configuration_subsystem/navigator_post_processor.py

```diff
@@ -223,12 +251,12 @@ def execution_environment_image(
             entry.value.current = f"{entry.value.current}:latest"
         return messages, exit_messages
 
-    @staticmethod
     @_post_processor
     def execution_environment_volume_mounts(
-        entry: Entry, config: ApplicationConfiguration
+        self, entry: Entry, config: ApplicationConfiguration
     ) -> PostProcessorReturn:
         # pylint: disable=unused-argument
+        # pylint: disable=too-many-branches
```

Comment:
Any chance to fix this?

Reply:
I think this would probably get fixed in a follow on PR that takes advantage of the logic in the new dataclass and replaces some of the logic here.

Even if that refactoring was in this PR, I think it would muddy the waters and distract from the new logic of delayed and normal.

Reply:
Yeah, it could be filed as a refactoring issue with ideas so it's not lost.